tests/coreutils/date.rs

```diff
@@ -0,0 +1,712 @@
+use std::ffi::OsString;
+use uutils_args::{Arguments, Options, Value};
+
+// Note: "+%s"-style format options aren't covered here, but should be!
+
+// +%s
+//   -I[FMT], --iso-8601[=FMT]  output date/time in ISO 8601 format.
+//   -R, --rfc-email            output date and time in RFC 5322 format.
+//       --rfc-3339=FMT         output date/time in RFC 3339 format.
+// date, hours, minutes, seconds, ns
+// date, seconds, ns
+
+#[derive(Default, Debug, PartialEq, Eq, Value)]
+enum Iso8601Format {
+    #[default]
+    #[value("date")]
+    // TODO: Express the concept "accepts prefixes" more nicely.
```

Comment:
I thought I built this into `Value` already? It's working here for example:

https://github.com/uutils/uutils-args/blob/a3ecc624a93ebd592a885567162f8cebb8fa4178/tests/coreutils/ls.rs#L480